### PR TITLE
breadcrumb back for dataset and folder and subfolder

### PR DESCRIPTION
**Pull request by tcnichol**
The breadcrumb is now back for datasets, folders, and subfolders.

### CODE REVIEW

File: frontend/src/components/datasets/Dataset.tsx [hunk 1]
```diff
@@ -49,7 +50,7 @@ export const Dataset = (): JSX.Element => {
 	// search parameters
 	const [searchParams] = useSearchParams();
 	const folderId = searchParams.get("folder");
-
+	console.log('folder id is', folderId);
```
Comment (longshuicy) [inline frontend/src/components/datasets/Dataset.tsx:53]: could you remove the console log?

File: frontend/src/components/datasets/Dataset.tsx [hunk 2]
```diff
@@ -280,6 +281,13 @@ export const Dataset = (): JSX.Element => {
 							disabled={false}
 						/>
 					</Tabs>
+					{folderId !== null ?
+							<Box>
```
Comment (longshuicy) [inline frontend/src/components/datasets/Dataset.tsx:285]: Would aligning with the file table will make it look cleaner? Thoughts?

image


Add 
```
 padding: "24px 24px 0 24px"
```

**Review by longshuicy — CHANGES_REQUESTED** [review]

File: frontend/src/components/datasets/Dataset.tsx (hunk 2 above)
Comment (tcnichol) [reply frontend/src/components/datasets/Dataset.tsx]: @longshuicy 

I added this but somehow it's not aligned properly, but it looks like it's okay on the Files component. Not sure what I did wrong.

File: frontend/src/components/datasets/Dataset.tsx (hunk 1 above)
Comment (tcnichol) [reply frontend/src/components/datasets/Dataset.tsx]: removed

File: frontend/src/components/files/File.tsx [hunk 3]
```diff
@@ -308,6 +307,7 @@ export const File = (): JSX.Element => {
 							disabled={false}
 						/>
 					</Tabs>
+					<MainBreadcrumbs paths={paths} />
```
Comment (longshuicy) [inline frontend/src/components/files/File.tsx:310]: you need to add the same thing here 
```
<Box style={{padding: "24px 24px 0 24px"}}>
```

**Review by longshuicy — APPROVED** [review]
Approves.

**Review by max-zilla — APPROVED** [review]
looks good.

**Pull request merged**